css/custom.css

```diff
@@ -100,6 +100,7 @@ section a p {
 	color: #fff;
 	/* better readability on bright background */
 	text-shadow: 0 0 2px rgba(10, 30, 20, .4);
+    margin-top: 10px;
```

Comment:
The indentation is off at some points in the file. The project uses tabs everywhere. :)

@camilasan do you want to adjust it? Love the change otherwise!

Reply:
Sorry, wrong editor :) Please, check again.